regolith/export.go

```diff
@@ -10,10 +10,10 @@ import (
 // resource pack based on exportTarget (a structure with data related to
 // export settings) and the name of the project.
 func GetExportPaths(
-	exportTarget ExportTarget, name string,
+	exportTarget ExportTarget, name string, WslUser string,
 ) (bpPath string, rpPath string, err error) {
 	if exportTarget.Target == "development" {
-		comMojang, err := FindMojangDir()
+		comMojang, err := FindMojangDir(WslUser)
```

Comment:
This line breaks compilation on Windows because the function signature on Windows doesn't take any arguments. The windows version of the function should take string argument as well even if it doesn't use it.

https://github.com/willster277/regolith/blob/59125f1632e53b209fcbaf1c7faf3f515e1c2bc2/regolith/compatibility_windows.go#L142-L158

Reply:
This doesn't really matter when we go with the solution that I suggested in the next comment.